### DISCUSSION TITLE
关于antd5中Tree组件中会有itemHeight属性

### DISCUSSION THREAD

**Discussion by ysl-ccc** [ant-design/ant-design#51277] [Q&A] [answered]
Tree组件中会有itemHeight属性，但是在官方文档api中没有itemHeight属性，请问一下itemHeight属性的作用，我发现设置还是不设置itemHeight都没影响。

**Comment by zombieJ** [comment, marked as answer]
`itemHeight` 用于虚拟滚动的预设最小高度，在 antd 中默认使用 Design Token 的 `controlHeight`。当使用虚拟滚动时，一个屏幕下需要渲染的条目数是 `listHeight / itemHeight`，这个值设置很多会导致虚拟滚动下渲染不全，很少则会过多渲染。

对于用户本身而言，这个属性不需要关心。因而也不作文档透出。

**Reply by ysl-ccc** [reply]
我之前以为itemHeight是由于使用自定义渲染结点titleRender，在titleRender中返回的ReactNode的高度（例如：80px），则需要设置itemHeight的值为80。
